src/sparseml/exporters/transforms/convert_quantizable_conv_integer.py

```diff
@@ -180,31 +125,32 @@ class ConvertQuantizableConvInteger(BaseTransform):
 
     def _transform(self, model: ModelProto) -> ModelProto:
         count_converted_nodes = 0
-        conv_nodes_indices, conv_nodes = zip(
-            *[
-                (idx, node)
-                for idx, node in enumerate(model.graph.node)
-                if node.op_type == "Conv"
-            ]
-        )
-        for idx, conv_node in zip(conv_nodes_indices, conv_nodes):
-            # Check whether there is a match
-            result = is_quantizable_conv_int(conv_node, model)
-            if result is None:
-                continue
-            input_quantize_node, weight_quantize_node, weight_dequantize_node = result
-            _LOGGER.debug(f"Matched quantizable Conv weight and bias: {conv_node.name}")
-            model = convert_conv_to_quantized(
-                model,
-                conv_node,
-                weight_quantize_node,
-                weight_dequantize_node,
-                input_quantize_node,
+        graph = ONNXGraph(model)
+        for match in iter_structural_matches(
+            graph,
+            parent_ops=[
```

Comment:
Ok, this is seriously awesome. I like how this one function exhaustively describes the condition for the match.

Reply:
But yeah, having a list of list called `parent_ops` needs a good documentation and comments :)

Reply:
Agree, i find it way more clear! And yeah, any suggestions on naming of anything? A good name will probably save us a lot of headache. Will add docstring now